clang/tools/scan-build-py/lib/libscanbuild/arguments.py

```diff
@@ -469,13 +469,13 @@ def parser_build_log(parser):
     parser.add_argument(
         '--parse-build-log',
         metavar='<file>',
-        help="""Specifies the build log file path of project specified by --in-root.""")
+        help="""Specifies the build log file path of project specified by --work-directory.""")
 
-def parser_in_root(parser):
+def parser_work_directory(parser):
     parser.add_argument(
-        '--in-root',
+        '--work-directory',
         metavar='<path>',
-        help="""Specifies the directory path for the root of the source tree that needs to be migrated.""")
+        help="""Specifies the work directory of the source tree that needs to be migrated.""")
```

Comment:
how about use "Specifies the work directory of the build log."

Reply:
work directory of source file in build log may be changed, I advice"Specifies the work directory of the source file in the build log that needs to be migrated."